general/1_terminal/task3-finiak.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+read -p  'enter text: ' name
+
+day=$(echo $name | awk '{print $NF}')
+NUMBER=$(echo $name | tr -dc '0-9')
+
+if [ $day == 'Sunday' ] || [ $day == 'Saturday' ] || [ $day == 'Неділя' ] || [ $day == 'Субота' ] || [ $day == 'Воскресенье' ] || [ $day == 'Суббота' ] || [ $day == 'रविवार' ] || [ $day == 'शनिवार' ]
+then 
+	echo "Looks like $NUMBER is a weekend day" 
+elif [ $day == 'Monday' ] || [ $day == 'Tuesday' ] || [ $day == 'Wednesday' ] || [ $day == 'Thursday' ] || [ $day == 'Friday' ] || [ $day == 'Понеділок' ] || [ $day == 'Вівторок' ] || [ $day == 'Середа' ] || [ $day == 'Четвер' ] || [ $day == "П'ятниця" ] || [ $day == 'Понедельник' ] || [ $day == 'Вторник' ] || [ $day == 'Среда' ] || [ $day == 'Четверг' ] || [ $day == 'Пятница' ] || [ $day == 'सोमवार' ] || [ $day == 'मंगलवार' ] || [ $day == 'बुधवार' ] || [ $day == 'गुरूवार' ] || [ $day == 'शुक्रवार' ]
+then
+	echo "Looks like $NUMBER is a working day"
+else
+	echo "Sorry the line you entered is incorrect"
```

Comment:
Let's step back a little bit, because this is a good exercise.

Me, the customer, told you to add support for Hindu and Russian, and you did that.

But let's say tomorrow I ask you to support German, Spanish and French, and you'll look up the names of the weekdays and do that too.

Then next week I ask for 20 more languages. At some point, you are going to get tired of adding all the names in different languages.

What if I told you there was a way to avoid changing it at all, and this way supports ALL of the languages, even the ones not invented yet.

Hint: you won't need string comparison in your solution.